tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -2815,6 +2815,51 @@ TEST_F(MatmulSchedulerPluginTest, BasicMatmul) {
       executor_cache.fusion(), outputs, {t0, t1}, {tref}, __LINE__, __FILE__);
 }
 
+// Test that we can segment a fusion that has a MatmulOp with epilogue
+// TODO: Once we can control the ExprEval and Matmul schedulers via options, run
+// this test with all three combinations (with and without each scheduler, but
+// at least one enabled).
```

Comment:
A version of this test that works like this TODO suggests is here (passing on that branch): https://github.com/NVIDIA/Fuser/pull/2236/files#diff-79605590906bd352b3f5d8798d0f16a00350ab8e740cdc0e64a89a7f8b34d204R341-R351